crates/trippy-tui/src/print.rs

```diff
@@ -32,7 +34,11 @@ pub fn print_man_page() -> anyhow::Result<()> {
 }
 
 pub fn print_locales() {
-    println!("TUI locales: {}", available_locales().iter().join(", "));
+    let locales = LANGUAGE_LOADER
+        .get_or_init(|| fluent_language_loader!())
```

Comment:
@nc7s modified to work with `OnceLock`.

Reply:
aside: we may want to push this impl into the locale `module` to avoid exposing `LANGUAGE_LOADER` (we can make it `doc(hidden)` perhaps since it must be public).